repos/spack_repo/builtin/packages/py_gcovr/package.py

```diff
@@ -26,6 +28,10 @@ class PyGcovr(PythonPackage):
     depends_on("python@2.7:2,3.5:", when="@:4", type=("build", "run"))
     depends_on("py-setuptools", type=("build", "run"))
```

Comment:
```suggestion
    depends_on("py-setuptools", type=("build", "run"), when="@:8.2")
```

We'll also want to constrain the past dependency on setuptools so that we don't keep pulling it in for newer builds.